### ISSUE TITLE
Is SIDE (Selenium IDE) dead ? 

### ISSUE THREAD

**Issue by AlphonseProffit** [SeleniumHQ/selenium-ide#1872]
Hi

there is more than 400 issues on Selenium IDE

It seems there is no more activity ...

What's the problem ?

Can you share some visibility ? 

Thanks

**Comment by agilare** [comment]
Indeed, many users of this important tool would be grateful, I think, to have some news, even small ones.

**Comment by toddtarsi** [comment]
Yeah, hey, I am the sole maintainer on Selenium IDE v4. It's effectively dead. I use it at my job, I have a plugin that does AI stuff too, but I also am fine with runtime exceptions, state caching issues, etc.

Times have changed, the world has changed. Companies used to budget for supporting OSS rather than just grinding out features nonstop. I don't have the personal bandwidth to do this at night anymore. I'm sorry, I wish you all the best, but it can't be me, and it isn't anyone else.

**Comment by agilare** [comment]
Oh, what a shame. Thanks for replying anyway
Sadly, a quality project that appears to have been widely adopted has been abandoned due to a lack of support. As often happens with open source, the project remains "frozen," and the well-designed documentation (https://www.selenium.dev/selenium-ide/), which was previously up-to-date, suggests that the project is still alive.
The cessation of development is understandable, of course, but it's a shame that the abandonment is total and a little time isn't spent informing users who have adopted the solution, such as a simple announcement, that would clarify the situation.
In any case, thank you for your work developing this great tool.

**Comment by darkartswizard** [comment]
Hey Todd,

I have been admiring your Selenium IDE project for many years, often
promoting it and occasionally contributing my suggestions.

It was always refreshing to see the messages pop across on my phone
with updates and feature requests.

I have hope it will be back in some form, but in the meantime it was a
great ride. I wish you and your team success in your next ventures.

Cheers!

[image: image.png]

**Comment by Joey-339** [comment]
This is sad news for most of the Selenium IDE users, we hope Selenium IDE can be continue

**Comment by vitalets** [comment]
Could anybody suggest any alternatives?

**Comment by dwarwick** [comment]
Playwright. We switched to Playwright and it works really well for us. I should caveat my suggestion though. Playwright is not a browser extension though. It is an alternative to Selenium. You have to write code in JavaScript, typescript, .Net, etc. Several languages supported.

**Comment by dwarwick** [comment]
We switched to Playwright, and it works really well for us.

**Comment by philRVT** [comment]
> Yeah, hey, I am the sole maintainer on Selenium IDE v4. It's effectively dead. I use it at my job, I have a plugin that does AI stuff too, but I also am fine with runtime exceptions, state caching issues, etc.
> 
> Times have changed, the world has changed. Companies used to budget for supporting OSS rather than just grinding out features nonstop. I don't have the personal bandwidth to do this at night anymore. I'm sorry, I wish you all the best, but it can't be me, and it isn't anyone else.

Hi @toddtarsi fully understand.  Please don't beat yourself up.  I've [submitted a PR](https://github.com/SeleniumHQ/selenium-ide/pull/1948) to alert people that -- [per Michael Mintz on Slack](https://seleniumhq.slack.com/archives/C0ABCS03F/p1775240986283049) -- people should migrate to [Selenium Base](https://github.com/seleniumbase/SeleniumBase) and use the Recorder in that.  Please review and accept? Thank you!

**Comment by rpallavisharma** [comment]
the last i checked, selenium ide worked for firefox. there was a blog @titusfortner wanted to bring out about selenium ide project, and mentioning, project is looking for maintainers, not sure about the latest update on this... seleniumbase is a great alternative, and maybe there exists other open source projects/ recorders which could also be listed. i believe the project leadership will have more information on this.. i remember it being discussed in tlc slack channel.. 
@toddtarsi .. thank you for the work for selenium ide .. helped me immensely in various projects... over many years.. and continues..
